cmd/lora-app-server/cmd/configfile.go

```diff
@@ -129,6 +129,15 @@ id="{{ .ApplicationServer.ID }}"
   status_topic_template="{{ .ApplicationServer.Integration.MQTT.StatusTopicTemplate }}"
   location_topic_template="{{ .ApplicationServer.Integration.MQTT.LocationTopicTemplate }}"
 
+  # Retained Messages
```

Comment:
Do you mind changing this to:

```
  # Retained messages configuration.
  #
  # The MQTT broker will store the last publised message, when retained message is set
  # to true. When a client subscribes to a topic with retained message set to true, it will
  # always receive the last published message.
  uplink_retained_message={{ .ApplicationServer.Integration.MQTT.UplinkRetainedMessage }}
  join_retained_message={{ .ApplicationServer.Integration.MQTT.JoinRetainedMessage }}
  ack_retained_message={{ .ApplicationServer.Integration.MQTT.ACKRetainedMessage }}
  error_retained_message={{ .ApplicationServer.Integration.MQTT.ErrorRetainedMessage }}
  status_retained_message={{ .ApplicationServer.Integration.MQTT.StatusRetainedMessage }}
  location_retained_message={{ .ApplicationServer.Integration.MQTT.LocationRetainedMessage }}
```

Then the values are automatically rendered.